### PR TITLE
Unify hosts format along the doc

### LINKED ISSUE #175: Unify server parameters in doc
In the [HTTP documentation](https://docs.plasmabio.org/en/latest/install/https.html), we advise to provide server parameters in the following format:
```
[servers]
51.178.95.237 name_server=dev1.plasmabio.org letsencrypt_email=contact@plasmabio.org
51.178.95.238 name_server=dev2.plasmabio.org letsencrypt_email=contact@plasmabio.org

[servers:vars]
ansible_python_interpreter=/usr/bin/python3
```

I would prefer we keep server parameters as one parameter per line with the following format:
```
[server1]
51.178.95.237 

[server2]
51.178.95.238

[server1:vars]
ansible_python_interpreter=/usr/bin/python3
name_server=dev1.plasmabio.org
letsencrypt_email=contact@plasmabio.org

[server2:vars]
ansible_python_interpreter=/usr/bin/python3
name_server=dev2.plasmabio.org
letsencrypt_email=contact@plasmabio.org
```

I propose to update the documentation.

### PR DESCRIPTION
**Pull request by pierrepo**
- [x] Add / update the documentation

Fix #175 

Documentation [Deploying with Ansible](https://docs.plasmabio.org/en/latest/install/ansible.html) is also modified:

- The case of multiple servers is addressed.
- The command line to use a single server from a multiple server `hosts` file is  provided:
    ```
    ansible-playbook site.yml -i hosts -u ubuntu --limit server1
    ```

### CODE REVIEW

**Comment by pierrepo** [comment]
Oups, by the way, this ``hosts`` format *works* since I'm using it for a while now :wink:

File: docs/install/ansible.rst [hunk 1]
```diff
@@ -270,8 +288,19 @@ The Ansible playbooks are located in the ``ansible/`` directory:
...
+
+
+Running playbook on a given server
+----------------------------------
+
+If you have multiple servers defined in the ``hosts`` file, you can run a playbook on a single server with the ``--limit`` option:
+
+.. code-block:: bash
+
+    ansible-playbook site.yml -i hosts -u ubuntu --limit server1
```
Comment (jtpio) [inline docs/install/ansible.rst:306]: If we use `-i` for the inventory, should we use `-l` instead of `--limit`?

Comment (pierrepo) [reply docs/install/ansible.rst:306]: You're perfectly right. I found sometime the character `l` a bit confusing. That's why I opted for the long form.

Comment (jtpio) [reply docs/install/ansible.rst:306]: Maybe we should keep `--limit` then? Or follow what the Ansible documentation uses?

Comment (pierrepo) [reply docs/install/ansible.rst:306]: Ansible documentation tends to use `--limit`:
https://docs.ansible.com/ansible/latest/user_guide/intro_patterns.html
> You can also limit the hosts you target on a particular run with the --limit flag:

Comment (jtpio) [reply docs/install/ansible.rst:306]: ok, then let's keep it like it is. Thanks for checking!

**Pull request merged**